Lombiq.Tests.UI/Extensions/ShortcutsUITestContextExtensions.cs

```diff
@@ -714,4 +715,23 @@ public static Task ExecuteJsonRecipeSiteSettingsAsync(this UITestContext context
     /// </summary>
     public static Task ExecuteJsonRecipeSiteSettingAsync<T>(this UITestContext context, T setting) =>
         context.ExecuteJsonRecipeSiteSettingsAsync(new Dictionary<string, object> { [typeof(T).Name] = setting });
+
+    /// <summary>
+    /// Enabled the "Shift Time - Shortcuts - Lombiq UI Testing Toolbox" feature directly.
+    /// </summary>
+    public static Task EnableTimeShiftingAsync(this UITestContext context) => context.EnableFeatureDirectlyAsync(ShiftTime);
+
+    /// <summary>
+    /// Sets the time shift to a specific value. If both <paramref name="days"/> and <paramref name="seconds"/> are
+    /// provided, then the <see cref="TimeSpan"/> values are added together.
+    /// </summary>
+    public static Task SetShiftTimeAsync(this UITestContext context, double days = 0, double seconds = 0) =>
```

Comment:
Any specific reason why these methods don't just accept `TimeSpan`, which is the type to handle such values, and the controller to just accept seconds? With `double`'s range, even that would allow setting 8,9864021011838002452723457144492e+67 millennia, so I think we're safe.

Reply:
I just passed the action's parameters as-is, without any further abstraction. But I agree, a TimeSpan overload would be cleaner for this extension method.

The controller supports both days and seconds, because I imagine you'd mostly want to shift whole days so that was my first choice. Using the days argument creates an easier to read navigation log as well, `ShiftTime/Set?days=30` is easier to understand than `ShiftTime/Set?seconds=2592000`. But in the off chance that you want to shift less than a day, fractional days are really unwieldy so I added seconds just for completion's sake.

Reply:
Is shifting time by whole days, seconds, and both added together so important of a use case to support it with separate overloads? Because this seems really strange to me. `TimeSpan` has an API for all this. I'd remove this basic arithmetic.

Why I'd use just seconds is to simplify the controller, and use seconds as a serialization of `TimeSpan` (what it supports with `TotalSeconds`).

Reply:
@sarahelsaig? And see https://github.com/Lombiq/UI-Testing-Toolbox/pull/418/files#r1835535385

Reply:
The important use-case is to have both options, for the trace to be more human readable as I mentioned in my previous comment. Adding them together is just the simplest implementation I could think of at the moment. In most cases it will be something plus zero, so an arithmetic operation is both easier and more efficient than a conditional operation.

Reply:
I understand the controller, then, though it still seems strange to me. How about the extension method overloads, though?